test/spec/utils/rest/post.spec.ts

```diff
@@ -28,30 +28,3 @@ test('Successfully POSTs to the given endpoint & stringifies body', async () =>
     },
   ]);
 });
-
-test('Successfully POSTs to the given endpoint and adds JSON header', async () => {
-  const fetchStub = jest.fn().mockImplementation(() => successRes);
-  (fetch as any) = fetchStub;
-
-  await expect(
-    post(
-      'https://example.com/hello/world',
-      API_KEY,
-      { public_address: '0x0123' },
-      { 'Content-Type': 'application/json' },
-    ),
-  ).resolves.toBe('hello world');
-
-  const fetchArguments = fetchStub.mock.calls[0];
-  expect(fetchArguments).toEqual([
-    'https://example.com/hello/world',
-    {
-      method: 'POST',
-      headers: {
-        'X-Magic-Secret-key': API_KEY,
-        'Content-Type': 'application/json',
-      },
-      body: '{"public_address":"0x0123"}',
-    },
-  ]);
-});
```

Comment:
Can we keep this? doesn't look NFT related

Reply:
I added this in a PR for minting, basically needed to add in the secret key header specifically for minting -- here's where it was added: https://github.com/magiclabs/magic-admin-js/pull/107/files#diff-1dafd08f79b2abad5cee6ab033b9a92fd6e01257c287673b6eb533a9c558335aR32

This test is for this change, which is only needed by minting: https://github.com/magiclabs/magic-admin-js/pull/108/files#diff-3ca6a15d564c2c26d46054f3bde8c43d1e5c79865143e55ad08164353ef58320L54

Reply:
Gotcha!